package.json

```diff
@@ -271,5 +271,7 @@
     "typescript": "^3.9.6",
     "vscode-test": "^1.6.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "axios": "^0.26.1"
```

Comment:
We only have dev dependencies here. Everything will be bundled in the end - so nothing here will be required at runtime later.